scenariolib/uabot.go

```diff
@@ -68,46 +68,51 @@ func (bot *uabot) Run() error {
 	}
 
 	count := 0
-	for {
-		// Run forever
+	for { // Run forever
+		select {
```

Comment:
well, kind of, because there wont be any message in the channel if not to quit.  so you dont want the go routine to stop on a <-